bucket/playback.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "1.7.0",
+    "description": "Playback is an application which allows you to play and manage your cartridges using Epilogue cartridge readers",
+    "homepage": "https://www.epilogue.co/",
+    "license": "Unknown",

```

Source code does not seem to be available:

```suggestion
    "license": "Freeware",
```